### DISCUSSION TITLE
Prepare for a job in tech with the new Career Readiness Experience!

### DISCUSSION THREAD

**Discussion by morganersery1** [community/community#47795] [GitHub Education]
![Career Readiness Twitter Card](https://user-images.githubusercontent.com/107881423/219716272-fa756d57-12e3-4b41-8f48-ddf3956a8f9e.png)

Check out the newest experience addition to the GitHub Global Campus and the Student Developer Pack—[Career Readiness](https://education.github.com/experiences/career_readiness)! Get job-search ready today.

### Wondering how to prepare for a career in tech?
The Career Readiness Experience takes you on a 4 module journey through different exercises designed to help you organize your skills and talents to launch a job, internship or Octernship search. We'll cover everything from identifying your career goals and building your personal brand to networking, preparing your application materials, and acing your interviews. 

Whether you're a new grad or a career changer, this experience will help you navigate the job search process confidently and land your next job.

### What can you do with the Career Readiness Experience?
The Career Readiness Experience launches 1 week at a time for 4 weeks starting 2/16/23. You can access them in the following order:

- Week 1: Starting 2/16
- Week 2: Starting 2/23
- Week 3: Starting 3/2
- Week 4: Starting 3/9

After 3/9/23 all four weeks will be available.

**Week 1:**
You will identify the skills you already have and which ones you need to learn. Review job descriptions, and from the list of our skills, we will identify if we have the skill, if we need to learn it, or if we already have it. You can use [AlgoExpert](https://www.algoexpert.io/product) and [Educative](https://www.educative.io/) for the skills you need to learn, which are included in the [student developer pack](https://education.github.com/pack).

**Week 2:** 
Building your professional assets: your resume, LinkedIn profile, GitHub profile, portfolio and cover letter. We’ll look at examples of successful professional assets and show you how you can leverage our pack partner, Visme’s templates to make a visually compelling CV. 

**Week 3:** 
How and where to search and apply for job openings including best practices on identifying companies, how to search for job vacancies and preparing for selection processes. Use our pack partner DailyBot to organize and track your search.

**Week 4**
How to prepare for interviews including knowing what the different types of interviews you may encounter, how to prepare answers for common questions and what kinds of skills to gain to best approach the technical interview process. For testing your technical interviewing skills complete the Interview Prep Course on Interview Cake using their Student Developer Pack offer. 

### Starting the Career Readiness Experience!
The [Career Readiness Experience](https://education.github.com/experiences/career_readiness) is part of the Student Developer Pack. Not a member yet? It’s available for all verified students ages 13+, and anywhere in the world where GitHub is available. [Join today](https://education.github.com/discount_requests/student_application) using your school-issued email, student ID or [other proof of academic enrollment](https://github.blog/2019-07-30-how-to-get-the-github-student-developer-pack-without-a-student-id/). Becoming a member allows you to learn how to use real-world development tools.

[Join the pack today!](https://education.github.com/discount_requests/student_application)

**Comment by martcpp** [comment]
How do I started

**Reply by morganersery1** [reply]
Hi @martcpp You can get started by going to the Experience landing page [here](https://education.github.com/experiences/career_readiness), scrolling to the bottom and select 'Week 1' in the **Get Career Ready** Resource.

**Comment by artwalker** [comment]
Hi, my week4 doesn't have content. Could you please help me?

**Comment by Kingkon963** [comment]
Hi,

I'm getting the following error while trying to accept the **Career-Readiness-Week-1** assignement,

"There was an error accepting the assignment. GitHub repository could not be created. Your instructor may need to allow forking of private repositories in the classroom's organization settings. If this problem persists, please have the Classroom admin contact support."

Who is my instructor? which private repository?
